### DISCUSSION TITLE
Support for dotted lines

### DISCUSSION THREAD

**Discussion by maluta** [tone-row/flowchart-fun#309] [Features / Syntax]
Is it possible apply dotted lines on flowcharts.fun? 

From ![Screen Shot 2022-06-22 at 18 17 15](https://user-images.githubusercontent.com/23658/175138607-c0a8cae8-b0dc-4ba5-8095-2504e4e59e2f.png) to (something like)
<img width="149" alt="Screen Shot 2022-06-22 at 18 17 22" src="https://user-images.githubusercontent.com/23658/175138620-253bf525-9f7e-4257-8921-922077f76ceb.png">

**Comment by rob-gordon** [comment]
@maluta Definitely! Here's an example showing how to style all edges, as well as how to target a single edge.

https://flowchart.fun/c#H4hQGcBcE8BsFMBcoAEKC0LzwQY0gPYBOiK8AJgObyppYwLJ12wCWAdvOlHEiuQUiQKtTNjyESZKvADasAIYAjHAF4A5GwDW8FJAAWrcOoC6tND0bmWHLpb7kF4fSJDBQAFUPgUCgA5+KADuxFo+StB60H4clLQAkuzk8OyQsSi4RPAKwj4KKGzsWnoEei4ZAK5EWakFtrQK7JHCAB6QpCoAZsS6+bgEsATsGVk58HkFyji0AMKjwr51RfysWfiwkRXg6Qa68C0K+JMqsHr7kNbaugZGpAAUXka+AcGh4c3RsQCU1rK4W4QALYoeIAERMKGI1jQEV85HI6UaKAApABWABCYLRoN8SRQWU68BquB2+hy0JsOjKtxQd3+UAIwLBXxQAHpWSgAJoECoZJEKWDgUpbXTbdiUBDoQq6fqAwEpSDgUCsgBUoChgIqsDS0tAsvlqSVoAAEgoAG66ToVdgAQhQgB4NwCR+6AVaygA

![dotted-lines](https://user-images.githubusercontent.com/159949/175144865-10814c4d-0f11-4ac5-ac48-50ba6f63ad49.png)

**Reply by maluta** [reply]
Thanks @rob-gordon, it worked perfectly!
